authority/provisioner/acme.go

```diff
@@ -22,7 +23,11 @@ type ACME struct {
 	// by clients when creating a new Account. If set to true, the provided
 	// EAB will be verified. If set to false and an EAB is provided, it is
 	// not verified. Defaults to false.
-	RequireEAB bool     `json:"requireEAB,omitempty"`
+	RequireEAB bool `json:"requireEAB,omitempty"`
+	// Challenges contains the enabled challenges for this provisioner. If this
+	// value is not set the default http-01, dns-01 and tls-alpn-01 challenges
+	// will be enabled, device-attest-01 will be disabled.
+	Challenges []string `json:"challenges,omitempty"`
```

Comment:
I remember there's some special `multiString` type somewhere that's a bit more flexible for users to configure. Does it make sense to use that here too?

Reply:
`multiString` is in a different package, and with custom types, we might need to use generics, which can be more complex.